Gemfile

```diff
@@ -10,6 +10,7 @@ gem 'pry', group: :development
 group :test do
   gem 'actionpack', '~> 6'
   gem 'activerecord', '~> 6'
+  gem 'ddtrace', '~> 0.51'
```

Comment:
I think `ddtrace` will be included in the dependencies as part of calling `gemspec` on L6. If so, I think we can remove this line from the `Gemfile`s.